.jenkins/common_job_properties.groovy

```diff
@@ -86,14 +86,19 @@ class common_job_properties {
   // Sets the pull request build trigger.
   static def setPullRequestBuildTrigger(def context,
                                         def commitStatusContext,
-                                        def successComment = '--none--') {
+                                        def successComment = '--none--',
+                                        def postCommitPhrase = '') {
     context.triggers {
       githubPullRequest {
         admins(['asfbot'])
         useGitHubHooks()
         orgWhitelist(['apache'])
         allowMembersOfWhitelistedOrgsAsAdmin()
         permitAll()
+        if (postCommitPhrase != '') {
+          triggerPhrase(postcommitPhrase)
```

Comment:
There could be cases where we might want to set the phrase but not trigger only. I agree there isn't such a case now, but current abstractions don't provide such an ability. Please don't change the abstractions, but add a comment to explain what's going on and why.